tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1283,8 +1283,9 @@ def _sub_fit_memory_save_wrapper(
             try:
                 _ds_ray = try_import_ray()
                 if not _ds_ray.is_initialized():
-                    _ds_ray.init(address="auto", logging_level=logging.ERROR, log_to_driver=False)
-            except:
+                    _ds_ray.init(logging_level=logging.ERROR, log_to_driver=False)
```

Comment:
Is there any downside to not specifying an `address` value vs setting to `"auto"`?

Reply:
In this setting, there is only an upside or, rather, avoid a failure case.

If `auto` is set, Ray assumes that there is an existing cluster and will throw an Error otherwise. While not providing the `address` results in creating a local Ray instance instead of throwing an error in that case. 

Generally, one should only use `auto` if we know that a cluster should exist or has existed before. In the edge cases described above, neither is the case.